hatchery/nextflow.go

```diff
@@ -660,6 +660,12 @@ func createBatchComputeEnvironment(nextflowGlobalConfig NextflowGlobalConfig, ne
 			return "", err
 		}
 
+		subnets := []*string{}
+		for _, subnet := range subnetids {
+			s := subnet
+			subnets = append(subnets, &s)
+		}
```

Comment:
Maybe add a comment that says that we're changing an array of strings to an array of string pointers because of such and such reason.